Holyhead-Harpies/cli-sprint2-jw
# added ability to add a payment type for active customer.

## Status
**READY**

## Migrations
NO

## Description
Adds logic for adding a payment type for the active customer. Also added menu page and class.

## Related Issues
#3 User Should be able to add a payment type for the active customer.

## Steps to Test or Reproduce

1.
```
git fetch origin add_payment_type_ml
git checkout add_payment_type_ml
ruby db/create_tables.rb
```
2.
After the database is created, because we do not have the logic for creating a customer yet, manually enter a record into the database with a CustomerId of 123

3. 
```
rspec spec/payment_type_controller_spec.rb
rspec spec/payment_type_model_spec.rb
``` 

All tests should pass.

4. Run:
```
ruby app/main.rb
```

5. Select option 3 and follow prompts. Once the record is successfully created, the user should return to the main menu with the message ```Payment Type added successfully.``` above the main menu.

## Impacted Areas in Application
List general components of the application that this PR will affect:

* Payment Types controller/model/test files/PaymentTypes table on database
